rafthttp/stream.go

```diff
@@ -392,45 +392,52 @@ func (cr *streamReader) dial(t streamType) (io.ReadCloser, error) {
 	lv := semver.Must(semver.NewVersion(version.Version))
 	if compareMajorMinorVersion(rv, lv) == -1 && !checkStreamSupport(rv, t) {
 		resp.Body.Close()
+		cr.picker.unreachable(u)
```

Comment:
i do not think we should mark this as unreachable. this is reachable, but the version is not compatible?